jdbi/src/main/java/io/micronaut/configuration/jdbi/transaction/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Classes specific to combining JOOQ with Spring JdbcTemplate.
+ *
+ * @author Dan Maas
+ * @since 1.3.1
```

Comment:
I have changed the version in master to `1.4.0` snapshot

Since this is a new feature it warrants a new minor release not a patch a release. Could you change this `@since` references to `1.4.0`